nrf-softdevice-macro/src/lib.rs

```diff
@@ -42,16 +44,26 @@ struct Characteristic {
 
 #[proc_macro_attribute]
 pub fn gatt_server(_args: TokenStream, item: TokenStream) -> TokenStream {
+
+    // Context for error reporting
+    #[cfg(not(feature = "nightly"))]
```

Comment:
We can unconditionally use the `Ctxt` for reporting errors, even on nightly. Makes the code simpler.

Reply:
got it